### PR TITLE
Adding Metronome job queue functionality to CLI

### PR DESCRIPTION
**Pull request by kensipe**
command:   `job queue`, `job queue startdeadline`, `job queue --json`, `job queue -q`

```
dcos job queue 
ID                         JOB RUN ID              
startdeadline  /startdeadline/201801172321181BlLt  
```

```
 dcos job queue --json
[
  {
    "jobId": "startdeadline",
    "runs": [
      {
        "runId": "/startdeadline/201801172321181BlLt"
      }
    ]
  }
]
```

Originally a PR on https://github.com/dcos/dcos-cli/pull/1133
with approvals

### CODE REVIEW

**Comment by kensipe** [comment]
run linux integration tests

**Comment by kensipe** [comment]
@bamarni moved the PR to this repo

File: python/lib/dcoscli/dcoscli/job/main.py [hunk 1]
```diff
@@ -228,6 +233,40 @@ def _remove(job_id, stop_current_job_runs=False):
...
+    :type quiet: bool
+    :returns: process return code
+    :rtype: int
+    """
+
+    client = metronome.create_client()
+    deployment_list = client.get_queued_job_runs(job_id)
+
+    if not deployment_list and not json_flag:
+        msg = "There are no deployments in the queue"
```
Comment (bamarni) [inline python/lib/dcoscli/dcoscli/job/main.py]: I wouldn't print a message to stdout here, empty output / table is fine. There is also the case where the quiet option is passed and the message might be treated as an ID.

Comment (kensipe) [reply python/lib/dcoscli/dcoscli/job/main.py]: yeah... this is consistent with other commands... but I'm with you... I prefer things to be silent if successful

File: python/lib/dcoscli/dcoscli/data/help/job.txt [hunk 2]
```diff
@@ -45,6 +46,8 @@ Commands:
         Updates a schedule on a job.
     job show runs
         Shows the successful and failure runs of a job.
+    job queue
+        Print a list of currently jobruns that a queued to launch
```
[the remark this answers is not in the document]
Comment (kensipe) [reply python/lib/dcoscli/dcoscli/data/help/job.txt]: I'll reword

**Comment by kensipe** [comment]
@bamarni all concerns addressed

**Comment by bamarni** [comment]
@mesosphere-ci run all integration tests

**Comment by kensipe** [comment]
haha.. tests are failing because I removed the output if nothing is in the queue.. and didn't change the test... I'll have an update tomorrow am.

**Comment by kensipe** [comment]
@mesosphere-ci run all integration tests

**Comment by kensipe** [comment]
@mesosphere-ci run all integration tests

**Comment by kensipe** [comment]
the failure on linux integration tests is something to do with metrics... and has nothing to do with this PR... is this a flaky test? or ?

**Pull request merged**